### PR TITLE
Fix Center Positioning of Mint/Redeem & Deposit/Withdraw Popup

### LINKED ISSUE #90: Mint popup doesn't center on the large screen
R9sjMXa6

### PR DESCRIPTION
**Pull request by math-marcellino**
Fix #90 
## Mint/Redeem Popup in Large Screen
image

## Deposit/Withdraw Popup in Large Screen
image

### CODE REVIEW

**Comment by pyk** [comment]
@math-marcellino btw, in mobile popup should be on the bottom

Screen Shot 2022-02-06 at 00 09 05
Screen Shot 2022-02-06 at 00 08 56

File: components/v1/MarketDetailPage/ButtonDepositOrWithdraw.tsx [hunk 1]
```diff
@@ -27,9 +27,9 @@ const ButtonDepositOrWithdraw: FunctionComponent<ButtonDepositOrWithdrawProps> =
         <Dialog.Root>
             <Dialog.Trigger className="w-full w-full rounded-full border border-blue-light-11 bg-blue-light-10 py-[11px] text-sm font-semibold leading-4 tracking-[-0.02em] text-gray-light-1 dark:border-blue-dark-11 dark:bg-blue-dark-10 dark:text-blue-light-1">Deposit or Withdraw</Dialog.Trigger>
             <Dialog.Overlay className="fixed inset-0 z-30 bg-gray-dark-1/60 backdrop-blur dark:bg-black/60" />
-            <Dialog.Content className="fixed left-0 bottom-0 z-30 w-screen sm:min-h-[477px] sm:-translate-y-1/3">
+            <Dialog.Content className="fixed left-0 bottom-0 z-30 flex h-screen w-screen items-center">
```
Comment (pyk) [inline components/v1/MarketDetailPage/ButtonDepositOrWithdraw.tsx]: maybe we can use `sm:items-center` instead?

Comment (math-marcellino) [reply components/v1/MarketDetailPage/ButtonDepositOrWithdraw.tsx]: Okay thank you for the review sir! I think it is fixed now..

**Comment by pyk** [comment]
Ty @math-marcellino!

**Pull request merged**